CMakeLists.txt

```diff
@@ -423,37 +423,30 @@ message(STATUS "BUILD_PLUGINS_DIR set to ${BUILD_PLUGINS_DIR}.")
 # OS specific checks
 TEST_BIG_ENDIAN(BIG_ENDIAN)
 
+# public if we use the headers of the dependency in our public headers
+set(libcsound_PUBLIC_LIBS "")
+
 ## HEADER/LIBRARY/OTHER CHECKS ##
 # Check if libsndfile is available, if not, disable build
-if(USE_LIBSNDFILE)
-    if(USE_VCPKG)
-        find_package(SndFile CONFIG REQUIRED)
-    else()
-        find_library(LIBSNDFILE_LIBRARY NAMES sndfile libsndfile-1 libsndfile FLAC ogg vorbis opus mp3lame mpg123)
-        if(NOT LIBSNDFILE_LIBRARY)
-            message(STATUS "libsndfile not found")
-            set(USE_LIBSNDFILE 0)
-        endif()
-    endif()
+if (USE_VCPKG)
+    find_package(SndFile CONFIG)
+else()
+    find_package(SndFile MODULE)
 endif()
+check_deps(USE_LIBSNDFILE SndFile_FOUND)
```

Comment:
You're not checking if the sndfile target was actually found in the VCPKG branch

Reply:
I'm not? The vcpkg find module seems to be setting SndFile_FOUND

Reply:
That's a good point. It's not a pattern we can rely on though afaik. Are config imports supposed to adhere to that standard as well? I'm not sure

Reply:
Well, the vcpkg config module shouldn't change much. But we can test for the existence of SndFile::sndfile instead I guess.

Reply:
That's what I mean. We can't assume any variables will be declared by the exported target but you can test the target directly as it's a cmake construct.

Reply:
Ok, changed to check for the target instead!